maxim/logger/livekit/agent_session.py

```
@@ -1,15 +1,26 @@
 import functools
 import inspect
 import traceback
+import time
+from typing import List 
 import uuid
 import weakref
 from datetime import datetime, timezone
 from io import BytesIO
 
 from livekit.agents import Agent, AgentSession
+from livekit.agents.metrics import LLMMetrics
```

Ruff (0.12.2)

`livekit.agents.metrics.LLMMetrics` imported but unused

Remove unused import: `livekit.agents.metrics.LLMMetrics`

(F401)